drivers/sensor/iis2iclx/iis2iclx_spi.c

```diff
@@ -50,24 +49,18 @@ static int iis2iclx_spi_read(struct iis2iclx_data *data, uint8_t reg_addr,
 		.count = 2
 	};
 
-
-	if (len > 64) {
```

Comment:
Why was this check removed?  Also below.

Reply:
If you want it removed, especially if it's in all ST sensor drivers, I'd rather see it in another PR.  Certainly it doesn't seem appropriate hidden in this commit.